src/views/Dashboard/Customer/Home/MyTest/TestResult.vue

```diff
@@ -0,0 +1,181 @@
+            //-       slot="icon"
+            //-       :icon="debioIcon"
+            //-       stroke
+            //-     )
+            
+            ui-debio-card(
+              class="mt-2"
+              tiny-card
+              with-icon
+              title="Rating"
+              sub-title="Help us improve your test experience by rating this service"
+              @click="actionRating"
+              )
+                ui-debio-icon(
+                  size="33"
+                  slot="icon"
+                  :icon="starIcon"
+                  stroke
+                  color="#c400a5"
+                )
+      //- Modal(
+      //-   :show="modalStatus"
+      //-   title="Please tell us about your experience!"
+      
+      //- )
+      ui-debio-modal(
+        :show="showModalRating"
+        :cta-action="submitRating"
+        title="Please tell us about your experience!"
+        cta-title="Submit"
+        @onClose="showModalRating = false"
+      )
+        template
+
+      ui-debio-modal(
+        :show="showModal"
+        :icon="checkCircleIcon"
+        :cta-action="closeModal"
+        :title="modalTitle"
+        cta-title="OK"
+        @onClose="showModal = false"
+      )
+
+</template>
+
+<script>
+import { downloadIcon, debioIcon, creditCardIcon, starIcon, checkCircleIcon } from "@/common/icons"
+import Modal from "@/common/components/Modal"
+export default {
+  name: "TestResult",
+  components: { Modal },
+  data: () => ({
+    downloadIcon,
+    debioIcon,
+    creditCardIcon,
+    starIcon,
+    checkCircleIcon,
+    resultLoading: false,
+    reportResult: "",
+    showModal: false,
+    showModalRating: false,
+    files: [
+      {
+        fileType: "report",
+        fileName: "DNA Sequence" + " Report",
+        fileLink: "https://ipfs.io/ipfs/QmdHkXZ2613Vx13dWQsvULsrDW5ggS31ahP4F3sNCKe6eD",
+        fileTitle: "Download Report",
+        fileSubTitle: "Download Your Test Report"
+      },
+      {
+        fileType: "result",
+        fileName: "DNA Sequence" + " Result",
+        fileLink: "https://ipfs.io/ipfs/QmRSQGh9zsjGnG2wQ2pLLX5gKZs1ufF1tp7aBgNAZmzJd9",
+        fileTitle: "Download Raw Data",
+        fileSubTitle: "Download Your Genomic Data"
+      }
+    ],
+    dummyResult: {
+      title: "Whole Genome Sequencing Test Report",
+      subTitle: "GSI Lab, 5 July 2021",
+      content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    }
+  }),
+
+  methods: {
+    actionDownload(type) {
+
+      console.log(type)
+    },
+
+    actionRating() {
+      console.log("rating")
```

Comment:
What does `actionDownload` do?

Why is there a `console.log` inside `actionRating`?

Reply:
this fucntion is for download result from ipsf, i think this page will be done by @rubenkristian